Public Record Office Victoria.js

```diff
@@ -0,0 +1,216 @@
+{
+	"translatorID": "99592877-f698-4e14-b541-b6181f6c577f",
+	"label": "Public Record Office Victoria",
+	"creator": "Tim Sherratt (tim@timsherratt.au)",
+	"target": "^https?://prov\\.vic\\.gov\\.au/(archive|search_journey)/*",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-01-21 03:13:11"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 Tim Sherratt
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+async function detectWeb(doc, url) {
+	if (getSearchResults(doc, true)) {
+		return "multiple";
+	// match items but not series, agencies, or functions (VPRS, VA or VF)
+	}
+	else if (url.match(/archive\/(?!VPRS|VA|VF)[A-Z0-9-]+/)) {
```

Comment:
use `test` for true/false tests on regex for efficiency.

Reply:
Thanks for the quick response! This is fixed.